### DISCUSSION TITLE
Spawn and Position a Window

### DISCUSSION THREAD

**Discussion by johnlindquist** [johnlindquist/kit#581] [Docs]
<meta path="help/spawn-and-position">
<meta description="show, getActiveScreen">

# Spawn and Position a Window

You can control the size/position of each `show` window you create, but you'll need some info from the current screen (especially with a multi-monitor setup!) to be able to position the window where you want it:

```js
let width = 480
let height = 320

let { workArea } = await getActiveScreen()
let { x, y, width: workAreaWidth } = workArea

show(
  md(`
# I'm in the top right of the current screen!

<div class="flex justify-center text-9xl">
😘
</div>
`),
  {
    width,
    height,
    x: x + workAreaWidth - width,
    y: y,
  }
)
```

**Comment by horacioh** [comment]
does this work with `arg` too?

**Reply by johnlindquist** [reply]
Do you mean position `arg` with x,y,width,height? Not currently. Is that something you'd like to see?

You can currently drag/resize any prompt and Kit.app will cache the size and position the next time it opens.
